p2p/net/conngater/conngater.go

```diff
@@ -51,7 +52,7 @@ func NewBasicConnectionGater(ds datastore.Datastore) (*BasicConnectionGater, err
 
 	if ds != nil {
 		cg.ds = namespace.Wrap(ds, datastore.NewKey(ns))
-		err := cg.loadRules()
+		err := cg.loadRules(context.TODO())
```

Comment:
I don't think we'll ever put a real context here, so maybe `context.Background()` is more appropriate?